.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: ruby
+rvm:
+  - 2.5.1
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+
+services:
+  - postgresql
+
+before_install:
+  - "echo 'gem: --no-document' > ~/.gemrc"
+  # Solves rainbow issue: https://github.com/sickill/rainbow/issues/48
+  - gem update --system
+
+before_script:
+  - cp config/database.travis.yml config/database.yml
+  - psql -c "DROP DATABASE IF EXISTS \"wbooksapi_test\";" -U postgres
```

Comment:
Why did you remove the User creation step?

Reply:
I removed the User creation step because I'm using the default PG user to access the test DB. 

In the rails-bootstrap project, the travis database config file is also using the default PG user so I think the user creation step is not necessary in that project. Find bellow the links to the travis config and travis db config files in the rails-bootstrap project.

https://github.com/Wolox/rails-bootstrap/blob/master/.travis.yml
https://github.com/Wolox/rails-bootstrap/blob/master/config/database.travis.yml